internal/util/util.go

```diff
@@ -101,3 +102,14 @@ func ExecuteCommandOutput(cmd *exec.Cmd, pathPrefix string, envars []string) str
 	}
 	return string(out)
 }
+
+// AppendProjectEnvToCmdEnv will add all the keys and values from envMap
+// into envList as key-value pair strings (e.g.: "key=value")
+func AppendProjectEnvToCmdEnv(envMap map[string]string, envList []string) []string {
```

Comment:
This will conflict with a change in David's PR since he also moved this function. But I agree it should be here in util.